Type conflict with @types/amplitude

Due to a namespace declaration there is currently a conflict in typing between @types/amplitude and this library. I see this has been removed in both #15 and #16 but am curious if there will be a released fix soon. This is preventing our application from building correctly.

See:
```
node_modules/@amplitude/experiment-js-client/dist/types/src/integration/amplitude.d.ts:5:9 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'amplitude' must be of type 'typeof import(".../node_modules/@types/amplitude-js/index")', but here has type 'any'.

5     var amplitude: any;
          ~~~~~~~~~

  node_modules/@types/amplitude-js/index.d.ts:8:1
    8 export as namespace amplitude;
      ~~~~~~
    'amplitude' was also declared here.
```

--- a/packages/browser/src/integration/amplitude.ts
+++ b/packages/browser/src/integration/amplitude.ts
@@ -14,12 +14,6 @@ type AmplitudeInstance = {
   options?: AmplitudeOptions;
   _ua?: AmplitudeUAParser;
   logEvent(eventName: string, properties: Record<string, string>): void;
-  _logEvent(
-    eventType: string,
-    eventProperties: Record<string, unknown>,
-    apiProperties: Record<string, unknown>,
-    userProperties: Record<string, unknown>,
-  );
   setUserProperties(userProperties: Record<string, unknown>): void;
   identify(identify: AmplitudeIdentify): void;
 };
@@ -84,7 +78,7 @@ export class AmplitudeUserProvider implements ExperimentUserProvider {
  * by the client (e.g. exposure).
  */
 export class AmplitudeAnalyticsProvider implements ExperimentAnalyticsProvider {
-  private amplitudeInstance: AmplitudeInstance;
+  private readonly amplitudeInstance: AmplitudeInstance;
   constructor(amplitudeInstance: AmplitudeInstance) {
     this.amplitudeInstance = amplitudeInstance;
   }
@@ -102,7 +96,7 @@ export class AmplitudeAnalyticsProvider implements ExperimentAnalyticsProvider {
 
   unsetUserProperty(event: ExperimentAnalyticsEvent): void {
     // if the variant does not have a value, unset the user property
-    this.amplitudeInstance._logEvent('$identify', null, null, {
+    this.amplitudeInstance['_logEvent']('$identify', null, null, {
       $unset: { [event.userProperty]: '-' },
     });
   }

